awx/api/serializers.py

```diff
@@ -2642,9 +2643,21 @@ class Meta:
 class JobRelaunchSerializer(JobSerializer):
 
     passwords_needed_to_start = serializers.SerializerMethodField()
-
-    class Meta:
-        fields = ('passwords_needed_to_start',)
+    retry_counts = serializers.SerializerMethodField()
+    hosts = serializers.ChoiceField(
+        required=False, allow_null=True, default='all',
+        choices=[
+            ('all', _('No change to job limit')),
+            ('failed', _('All failed and unreachable hosts')),
+            ('ok', _('Hosts with 1 or more ok tasks')),
+            ('changed', _('Hosts with 1 or more changed tasks')),
```

Comment:
@wenottingham are we going to awx "ok", "changed" as options in the API here?

Reply:
"ok", "failed"

Reply:
"all" is the default option that doesn't change the limit from the prior job. "ok" is a subset that doesn't include skipped hosts. From discussions up to this point, that would imply that the options we're surfacing are "all", "failed", and "unreachable"